package.json

```diff
@@ -136,7 +136,6 @@
 		"@automattic/calypso-build": "^9.0.0",
 		"@automattic/calypso-color-schemes": "^2.1.1",
 		"@automattic/calypso-doctor": "^0.1.0",
-		"@automattic/calypso-polyfills": "^1.0.0",
```

Comment:
This dep was not needed, `client/` already had a dependency on `calypso-polyfills`